### DISCUSSION TITLE
Issues with some older browser versions

### DISCUSSION THREAD

**Discussion by floflor** [nextauthjs/next-auth#2165] [Help]
Hello everybody, I'm having some issues... 
Appearently nextAuth doesn't work in some older versions of Chrome and Firefox. 

### Chrome

- Version 71 ✅ ⇒ status 200
- Version 70 ✅ ⇒ status 200
- Version 69 ✅ ⇒ status 200
- Version 68 ✅ ⇒ status 200
- Version 67 ⚠️ ⇒ status 304 (2018)
- Version 66 ⚠️ ⇒ status 304

### Firefox

- Version 69 ✅ ⇒ status 200
- Version 62 ✅ ⇒ status 200
- Version 61 ✅ ⇒ status 200
- Version 59 ⚠️ ⇒ status 304 (2018)
- Version 58 ⚠️ ⇒ status 304

We're having the problem at our user auth function.. 
Similar projects like the official demo and getting started have the same issue  (SEE => https://next-auth.js.org/getting-started/example )

We're currently implementing nextAuth in other apps and I would like to ask you if somebody has notice this before, because i've been searching but I couldn't find info about this.

**Comment by balazsorban44** [comment]
Interesting. We try to compile down to the same level as Next.js does:

https://www.github.com/nextauthjs/next-auth/tree/main/config%2Fbabel.config.js

Supporting anything lower seems unnecessary as the rest of your app wouldn't work anyway. I wonder what is kissing. Do you have an error message/repro?

**Reply by floflor** [reply]
Hi @balazsorban44 , thanks for your quick reply. 
We don't have any error log, but our app behaves the same as https://next-auth-example.vercel.app/
Anyway about what you mentioned, our Next app works perfectly in these versions, except for the login part.
We isolated the problem by removing the nextAuth layer (provider), also we checked the rest of the functionality and works perfect without the login.

**Reply by balazsorban44** [reply]
I see. It would be very nice if you could pin this down to some kind of a compilation error, so we could add a polyfill manually, if that is the problem here. I don't mind either if you opened a bug report about it as well.

This problem sounds very similar to https://github.com/nextauthjs/next-auth/issues/2018
